cime_config/cesm/machines/config_batch.xml

```diff
@@ -91,8 +91,8 @@
       <directive                       > -R "span[ptile={{ ptile }}]"</directive>
       <directive                       > -N  </directive>
       <directive default="poe"         > -a {{ poe }} </directive>
-      <directive default="cesm.stdout" > -o {{ cesm_stdout }}.%J  </directive>
-      <directive default="cesm.stderr" > -e {{ cesm_stderr }}.%J  </directive>
+      <directive default="cesm.stdout" > -o {{ output_error_path }}.%J  </directive>
```

Comment:
Should these both be "output_error_path"? I was expecting to see different values for output and errput.